qemu/tests/seabios.py

```diff
@@ -84,12 +91,6 @@ def reboot_check():
 
     error.context("Display and check the boot menu order", logging.info)
```

Comment:
This line can be a prompt for `vm.send_key(boot_menu_key)`, you could put it before line82.

Reply:
@sitoliu  boot menu order is displayed after vm.send_key.  so if put it before line82.  it doesn't works.